### ISSUE TITLE
Possibly a memory corruption

### ISSUE THREAD

**Issue by kjk** [sumatrapdfreader/sumatrapdf#3679]
Seen crash reports due to memory corruption in 3.5.15545 and 3.5.15510 but not before 3.5.15502 so likely introduced it between 3.5.15502 and 3.5.15510.

could be https://github.com/sumatrapdfreader/sumatrapdf/commit/ace666e473e1fb6a257de6323f9e1def329112ba fixed in https://github.com/sumatrapdfreader/sumatrapdf/commit/aabfa644bf1891814b3a3d07995c2480d35857ca

**Comment by kjk** [comment]
```
15502: 7683de3fe When add annotation. put focus in 'Contents' box
15510: a1f4a327d implement Ctrl + Backspace in Edit control (fixes #3502)
```
Unlikely::
* https://github.com/sumatrapdfreader/sumatrapdf/commit/7683de3fee4ff16109e15a9c299fa4c045662e72 
* https://github.com/sumatrapdfreader/sumatrapdf/commit/584fb5ebd196549e2d6529a76913a7e86e80802e
* https://github.com/sumatrapdfreader/sumatrapdf/commit/37dc3d215975257d3a6995a67fda6eb8340c5b6a
* https://github.com/sumatrapdfreader/sumatrapdf/commit/c80ed67300ba859e757dde04e1a81e834f89d128
* https://github.com/sumatrapdfreader/sumatrapdf/commit/8cbd361e900de1245052ce7d068a114d02db8f20
* https://github.com/sumatrapdfreader/sumatrapdf/commit/484f0f21b0f9fb58af1e0a2420a787b0faa3575e
* https://github.com/sumatrapdfreader/sumatrapdf/commit/a1f4a327d0643a645375481621321ccacaaf5727

Maybe:
* https://github.com/sumatrapdfreader/sumatrapdf/commit/5c63c533403e140ca8dc1051ef629a8166ad8ad2
* https://github.com/sumatrapdfreader/sumatrapdf/commit/a557e8fd81cbd52d21a12e956ddfc05302afff1e

**Comment by kjk** [comment]
Not seeing this crash anymore
